README.md

```diff
@@ -43,7 +46,7 @@ Parameters:
 
 ### verifier.verify
 
-This method will decode the token, verify the issuer, audience, expiration, algorithm and nonce claims and after that will verify the token signature.
+This method will decode the token then verify the token for OIDC compliance using a series of checks on the claims found in the token.
```

Comment:
What about linking to the spec? https://openid.net/specs/openid-connect-core-1_0-final.html#IDTokenValidation

Reply:
@lbalmaceda Done